run-build.sh

```diff
@@ -100,31 +98,19 @@ while [[ $# > 0 ]]; do
             LINUX_PORTABLE_INSTALL_ARGS="--runtime-id linux-x64"
             CUSTOM_BUILD_ARGS="/p:Rid=\"linux-x64\" /p:OSName=\"linux\" /p:IslinuxPortable=\"true\""
             ;;
-        --all-linux-installers)
-            ALL_LINUX_INSTALLERS_TARGET="/t:BuildAndPublishAllLinuxDistrosNativeInstallers"
-            ;;
-        --generate-installers)
-            GENERATE_INSTALLERS_TARGET="/t:GenerateInstallersAndCopyOutOfSandBox"
-            ;;
         --stage0)
             STAGE0_SOURCE_DIR=$2
             shift
             ;;
         --help)
-            echo "Usage: $0 [--configuration <CONFIGURATION>] [--skip-prereqs] [--nopackage] [--docker <IMAGENAME>] [--help]"
+            echo "Usage: $0 [--configuration <CONFIGURATION>] [--architecture <ARCHITECTURE>] [--skip-prereqs] [--nopackage] [--nobuild ] [--help]"
             echo ""
             echo "Options:"
             echo "  --configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
+            echo "  --architecture <ARCHITECTURE>       Build the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
             echo "  --skip-prereqs                      Skip checks for pre-reqs in dotnet_install"
             echo "  --nopackage                         Skip packaging targets"
             echo "  --nobuild                           Skip building, showing the command that would be used to build"
-            echo "  --docker <IMAGENAME>                Build in Docker using the Dockerfile located in scripts/docker/IMAGENAME"
```

Comment:
Could you please explain why have you removed the usage documentation for the options for building in docker container and for the `--linux-portable` `--stage0` option? All of these options are still supported by the script and they are needed e.g. for Alpine, RHEL 6 or for bootstrapping new Unixes in the future.

Reply:
"linux-portable" is used only by our Linux VSO build and as such should be an unadvertised parameter.
@janvorli 
Do you feel that our customers are using the "docker" and "stage0" parameters?

Reply:
It seems strange to not to report all parameters that the script can get. In all of our other repos, we list all parameters, no matter how obscure they are. 
Anyone from the community can attempt to bring up the CLI for a Linux repo or Unix flavor of their preference and in such case, they would need the stage0 parameter and some would need the docker one.

Reply:
@janvorli 
I have put back the 'docker' and 'stage0' parameters in the usage.